MediaGalleryMetadata/Model/File/ExtractMetadata.php

```diff
@@ -72,12 +81,22 @@ public function execute(string $path): MetadataInterface
     private function getEmptyResult(): MetadataInterface
     {
         return $this->metadataFactory->create([
-            'title' => '',
-            'description' => '',
-            'keywords' => []
+            'title' => null,
```

Comment:
Let's declare the Metadata constructor parameters null by default to avoid specifying them explicitly when there is no value